data.csv

```diff
@@ -0,0 +1,22 @@
+Proto_list,Blacklist,Banwords
+#netsec,Pornhub,porn
+#DNS,xnxx,pornhub
+#DANE,,xvideo
+#DNSSEC,,sex
+#QUIC,,
```

Comment:
What does the double coma signify?

Reply:
That's how a CSV file is written. As the full form says, **C**omma **S**eparated **V**alues :)